contents/blog/how-to-run-transparent-company.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2022-07-26
+title: "How to run a transparent startup"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+There really hasn't been a single situation where sharing pretty much everything we do hasn't been to our company's advantage.
+
+Most companies seem to share when necessary but not by default. I hope this piece changes that - as a startup, you've done something wrong if you don't trust your own team.
+
+Being realistic, to help this most people - this is mainly written about _internal_ transparency... and it's startup-focussed too, since that's what I know.
+
+If you want to gain even more advantages here, I would encourage you though to share this stuff by default externally unless there's a good reason not to.
+
+There are probably some startups where this doesn't work. Perhaps in design led companies, where control might be a competitive advantage and is more important than autonomy, or in companies that aren't managed consistently - you can't share a non-existent pay structure, for example (and, yes, it is hard work to implement something like that from very early, but we felt that work was worth it to prevent unfairness and employee churn).
+
+## Why this matters
+
+This section is a bit effusive. Skip it if you're not in the mood for that.
+
+I _cannot believe_ that more companies, and especially startups, don't do this stuff. It is in the top five most impactful decisions we've made.
+
+Above all, it enables people to work autonomously. There are two ways to grow a team - autonomy, or control. We chose the former because it's the kind of place we'd want to work, and therefore it makes it a better place to work for our team. That lets us hire and retain stronger people. Talent compounds, y'all.
+
+Transparency works particularly well in an all remote company. Like ours. That's because having this stuff written down and shared makes asynchronous work much easier.
+
+We didn't expect this, but hiring great has become much easier. Since we share most of the below publicly, people thinking about working here can build trust much more easily, and it makes us stand apart from the thousands of other startups. Transparency makes the whole company a joint project, between our team and the community around it too.
+
+## General communication
```

Comment:
Main bit of feedback from me is that I don't really get this structure or subheadings from here onwards. General communication isn't something we're transparent about, for example, nor is it a useful bucket for some of the contained topics (e.g. tool minimalism). I don't think `People` is a good subheading either, as it makes the actual content of these sections (e.g. the actual list of what we're transparent about) harder to locate in the article. 

Personally, I'd just restructure some of this content so that this section is called 'What we're transparent about'. I'd use the topic of tool minimalism as an intro, then I'd list all of the things we're transparent about as directly as possible: Fundraising, Hiring/Firing Decisions, Pay Structure, etc.